rpctypes/types.go

```diff
@@ -48,18 +48,20 @@ type EthSendBundleArgs struct {
 	MinTimestamp      *uint64         `json:"minTimestamp,omitempty"`
 	MaxTimestamp      *uint64         `json:"maxTimestamp,omitempty"`
 	RevertingTxHashes []common.Hash   `json:"revertingTxHashes,omitempty"`
-	ReplacementUUID   *string         `json:"replacementUuid,omitempty"`
+	ReplacementUUID   *uuid.UUID      `json:"replacementUuid,omitempty"`
 	Version           *string         `json:"version,omitempty"`
 
 	ReplacementNonce *uint64         `json:"replacementNonce,omitempty"`
 	SigningAddress   *common.Address `json:"signingAddress,omitempty"` // may or may not be respected depending on the context
 	RefundIdentity   *common.Address `json:"refundIdentity,omitempty"` // metadata field to improve redistribution ux
 
 	DroppingTxHashes []common.Hash   `json:"droppingTxHashes,omitempty"`
-	UUID             *string         `json:"uuid,omitempty"`
+	UUID             *uuid.UUID      `json:"uuid,omitempty"`
 	RefundPercent    *uint64         `json:"refundPercent,omitempty"`
 	RefundRecipient  *common.Address `json:"refundRecipient,omitempty"`
 	RefundTxHashes   []string        `json:"refundTxHashes,omitempty"`
+	// if set to true refund will be paid out of block
```

Comment:
nit: maybe move comment to right of code like some of the above fields, which would also make the lines align 🙏